lmdeploy/pytorch/engine/logits_process.py

```diff
@@ -224,17 +240,17 @@ def __init__(self, sampling_inputs: SamplingInputs,
         self.sampling_inputs: SamplingInputs = sampling_inputs
         self.ignore_eos = ignore_eos
 
-    def __call__(self, input_ids: torch.LongTensor,
-                 scores: torch.FloatTensor) -> torch.FloatTensor:
+    def __call__(self, scores: torch.FloatTensor,
```

Comment:
why change the sequence of scores and all_ids

Reply:
I did not change it, I just delete input_ids and put all_ids.